project/plugins.sbt

```diff
@@ -1,38 +1,23 @@
-scalaVersion := "2.10.4"
```

Comment:
In this plugins file, I removed a bunch of old plugins which I think were carryovers from `spark-csv` or something. This file now matches `spark-csv`'s current plugins file.